src/lib/radio/radio.ts

```diff
@@ -239,6 +239,9 @@ export class MdRadioButton implements OnInit {
   @HostBinding('class.md-radio-focused')
   _isFocused: boolean;
 
+  /** Tabindex for the input element. */
+  @Input() tabindex: number = 0;
```

Comment:
I don't remember right now, but I was testing against the local demo app. I'm retrying now and it works fine without the tabindex, so I assume it was because I wasn't seeing the ripple. I've removed  the tabindex changes from the PR.